client/src/sass/application.scss

```diff
@@ -145,7 +149,11 @@ label {
     padding-left: $not-expanded-horizontal-margins;
     padding-right: $not-expanded-horizontal-margins;
     position: fixed;
-    z-index: 1;
+    z-index: 200;
```

Comment:
please use the `z` function, to have z-indexes relative to https://github.com/Chocobozzz/PeerTube/blob/2e7f262724dd64a209e0bad5930ba29bb4f801c3/client/src/sass/include/_variables.scss#L123-L139